.changelog/24297.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+scheduler: fix a bug where we did not account for poststart tasks resources
```

Comment:
Minor suggestion to change the wording.
```suggestion
scheduler: Fixed a bug where resource calculation did not account correctly for poststart tasks
```

Reply:
Thanks @jrasell , I have amended your suggestion in the commit and changelog message, also rebased the branch !